File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/RacesByCapacityResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class RacesByCapacityResponse : IResponse
    {
        private readonly IEnumerable<Race> races;

        public RacesByCapacityResponse(IEnumerable<Race> races)
        {
            this.races = races;
        }

        public IEnumerable<Race> Races { get { return races; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab08/Java/networking/src/main/java/com/ubb/protocol/response/RacesByCapacityResponse.java
package com.ubb.protocol.response;

import com.ubb.model.Race;

import java.util.Collection;

public class RacesByCapacityResponse implements Response {
    private final Collection<Race> races;

    public RacesByCapacityResponse(Collection<Race> races) {
        this.races = races;
    }

    public Collection<Race> getRaces() {
        return races;
    }
}
